### DISCUSSION TITLE
求助各位，听歌排行是哪个接口呢？

### DISCUSSION THREAD

**Discussion by wood3n** [Binaryify/NeteaseCloudMusicApi#1660] [Q&A] [answered]
请问听歌排行是哪个接口呢？

**Comment by Revincx** [comment, marked as answer]
是 `/user/record`

参见[文档此处](https://neteasecloudmusicapi-docs.4everland.app/#/?id=%e8%8e%b7%e5%8f%96%e7%94%a8%e6%88%b7%e6%92%ad%e6%94%be%e8%ae%b0%e5%bd%95)


> 获取用户播放记录
> 
> 说明 : 登录后调用此接口 , 传入用户 id, 可获取用户播放记录
> 
> 必选参数 : `uid` : 用户 `id`
> 
> 可选参数 : `type` : `type=1` 时只返回 `weekData`, `type=0` 时返回 `allData`
> 
> 接口地址 : `/user/record`
> 
> 调用例子 : `/user/record?uid=32953014&type=1`

**Reply by wood3n** [reply]
感谢！
